src/encoded/types/dataset.py

```diff
@@ -77,7 +77,7 @@ class Dataset(Item):
     audit_inherit = [
         'original_files',
         'revoked_files',
-        'contributing_files'
+        'contributing_files',
```

Comment:
Weird this didn't have a comma before.  I did a mock test, I suspect the prod Dataset.audit_inherit list looks like
```
['original_files', 'revoked_files', 'contributing_filessubmitted_by', 'lab', 'award', 'documents.lab']
```

Reply:
yeah bug of irrelevance